packages/big-design/src/components/StatelessPagination/StatelessPagination.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  ArrowDropDownIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from '@bigcommerce/big-design-icons';
+import React, { memo, useMemo } from 'react';
+
+import { MarginProps } from '../../helpers';
+import { Dropdown, DropdownItem } from '../Dropdown';
+import { Flex, FlexItem } from '../Flex';
+
+import { StyledButton } from './styled';
+
+export interface StatelessPaginationLocalization {
+  previousPage?: string;
+  nextPage?: string;
+  rangeLabel?: string;
+  label?: string;
+}
+
+const getDefaultLocalization = (
+  itemsPerPage: number,
+): Required<StatelessPaginationLocalization> => ({
+  previousPage: 'Previous page',
+  nextPage: 'Next page',
+  label: 'pagination',
+  rangeLabel: `Show ${itemsPerPage} items`,
+});
+
+export interface StatelessPaginationProps extends MarginProps {
+  itemsPerPage: number;
+  itemsPerPageOptions: number[];
+  onItemsPerPageChange(range: number): void;
+  onNext?(): void;
+  onPrevious?(): void;
+  localization?: StatelessPaginationLocalization;
+}
+
+export const StatelessPagination: React.FC<StatelessPaginationProps> = memo(
+  ({
+    itemsPerPage,
+    itemsPerPageOptions,
+    onNext,
+    onPrevious,
+    onItemsPerPageChange,
+    localization: customLocalization,
+  }) => {
+    const localization: Required<StatelessPaginationLocalization> = useMemo(
+      () => ({ ...getDefaultLocalization(itemsPerPage), ...customLocalization }),
+      [customLocalization, itemsPerPage],
+    );
```

Comment:
Lot of hoop-jumping to handle default and custom localisation 😂

I imagine there aren't too many ways of simplifying this without compromises. The one thing I can think of is to go back to treating the default `rangeLabel` field as its own special case which would reduce the number of dependencies involved for memoisation, but now we're just backtracking. This is probably the best solution.